5.5/contrib/my.cnf.template

```diff
@@ -0,0 +1,29 @@
+[mysqld]
+user = mysql
+
+datadir = ${MYSQL_DATADIR}
+basedir = /opt/rh/mysql55/root/usr
+plugin-dir = /opt/rh/mysql55/root/usr/lib64/mysql/plugin
+
+general_log = ON
+
+# Disable unix socket
+socket =
+
+# Disabling symbolic-links is recommended to prevent assorted security risks
+symbolic-links = 0
+
+# http://www.percona.com/blog/2008/05/31/dns-achilles-heel-mysql-installation/
+skip_name_resolve
+
+#
+# Settings configured by the user
+#
+
+${MYSQL_TIME_ZONE}
```

Comment:
I still don't like this. We should use the env vars wisely to provide **values** to configuration directives... not to generate the directives themself. Can't you just add default-time-zone =${TZ} ? What happen when this setting will be just set to blank value? Is that default?